buidl-labs/crypto-code-school-inside-tezos
# Interface for Smartpy compiler.

**What does this PR do?**
Integrates an interface for the smartpy compiler

**Description of Task to be completed?**
- Interface to show compiled Michelson and Storage

**How should this be manually tested?**
Go through the chapters of module-2 and module-3. Look at the compiled code by clicking on "Show Compiled Code" that is present when you click "Check"
